infra/gcp/lib_gcs.sh

```diff
@@ -56,6 +56,28 @@ function ensure_public_gcs_bucket() {
     ensure_gcs_role_binding "${bucket}" "allUsers" "objectViewer"
 }
 
+# Set up logging
+# $1: The GCS bucket (e.g. gs://k8s-infra-foo)
+function ensure_gcs_bucket_logging() {
+    if [ $# != 2 ] || [ -z "$1" ]; then
```

Comment:
Still outstanding from last review
```suggestion
    if [ $# != 1 ] || [ -z "$1" ]; then
```

Reply:
All comments addressed, thank you